src/Symfony/Component/HttpFoundation/Session/Storage/NativeSessionStorage.php

```diff
@@ -367,17 +366,16 @@ public function setOptions(array $options)
      * @see http://php.net/sessionhandler
      * @see http://github.com/drak/NativeSession
      *
-     * @param AbstractProxy|NativeSessionHandler|\SessionHandlerInterface|null $saveHandler
+     * @param AbstractProxy|\SessionHandlerInterface|null $saveHandler
      *
      * @throws \InvalidArgumentException
      */
     public function setSaveHandler($saveHandler = null)
     {
         if (!$saveHandler instanceof AbstractProxy &&
-            !$saveHandler instanceof NativeSessionHandler &&
```

Comment:
should be removed as that's a BC break, isn't it?

Reply:
Nope, since Symfony 3.0, `NativeSessionHandler` always extends `\SessionHandler`. Thus now the rule `!$saveHandler instanceof NativeSessionHandler` is covered by `!$saveHandler instanceof \SessionHandlerInterface`